test/language/class/syntax/class-declaration-binding-identifier-class-element-list.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    ClassDeclaration:
+      class BindingIdentifier ClassTail
+
+    ClassTail:
+      ... { ClassBodyopt }
+
+    ClassBody[Yield] :
+      ClassElementList[?Yield]
+
+
+    ClassElementList[Yield] :
+      ClassElement[?Yield]
+      ClassElementList[?Yield] ClassElement[?Yield]
+
+    ClassElement[Yield] :
+      MethodDefinition[?Yield]
+      static MethodDefinition[?Yield]
+      ;
+
+
+---*/
+class A {
+  method() {}
+  static method() {}
+  ;
```

Comment:
It seems as weird as allowing backticks or commas or forward slashes or any other random punctuation mark.